ts/components/messages/MessageListComponent.tsx

```diff
@@ -62,7 +62,11 @@ class MessageListComponent extends React.Component<Props> {
 
   private FlatListRef = React.createRef<FlatList<MessageState>>();
   private scrollToTop = () => {
-    if (this.FlatListRef.current) {
+    if (
+      this.FlatListRef.current &&
+      this.FlatListRef.current.props.data &&
```

Comment:
this.props.messages?

Reply:
Ok, `this.props.messages` is fine as well. Finally the following check does the same as prev commit: 
```
    if (this.FlatListRef.current && this.props.messages.length) {
```